src/main/java/org/jitsi/impl/protocol/xmpp/extensions/ConferenceIq.java

```diff
@@ -297,7 +311,7 @@ public void setIdentity(String identity)
          */
         public Property()
         {
-            super(null, ELEMENT_NAME);
+            super(ConferenceIq.NAMESPACE, ELEMENT_NAME);
```

Comment:
maybe we should add a fixme in `AbstractPacketExtension` for changing the constructor to match the order of smack 4's (elementname, namespace) as opposed to the old (namespace, elementname)

Reply:
It seemed too dangerous switching two String parameters. I'll think about it.

Reply:
yeah agree...it's quite easy to get them wrong.  don't need to fix it in this pr, but wondered if a fixme comment would be helpful so eventually we can get everything consistent with smack.

Reply:
A feasible way might be renaming `AbstractPacketExtension` to `AbstractExtensionElement` and changing the constructor along the way.